### DISCUSSION TITLE
When mounting weed with multiple filer servers, does the client always select a random filer server from the pool for each file operation?

### DISCUSSION THREAD

**Discussion by CKHH** [seaweedfs/seaweedfs#5525] [Q&A]
Hello,

When mounting weed with multiple filer servers (i.e. `weed mount -filer=<server_1>,<server_ip_2>,<server_ip_3>,<server_ip_4>`...), does the client randomly select a filer server during any file operation, or does it consistently return to the same server for any file operation?

For example, I did several file listing attempts and copying various files around a few times, does each of the file operations select a random filer server or it always use a same server from the pool?

Thanks.

**Comment by janithcooray** [comment]
+1 
i have noticed it tends to start from the order of -filler and during network failures it switches to the next. this has caused the filers to be on inconsistent on selecting the filer servers despite being in the same Dc-rack.
